super_team_picker_4/index.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const app = express();
+
+const path = require('path')
+app.use(express.static(path.join(__dirname, 'public'))) //http://localhost:3000/public
+
+const methodOverride = require('method-override');
+
+const logger = require('morgan');
+app.use(logger('dev'));
+
+app.set('view engine', 'ejs');
+app.set('views', 'views');
+app.use(express.urlencoded({extended: true}))
+
+app.use(methodOverride((req, res)=> {
+  if(req.body && req.body._method) {
+    const method = req.body._method;
+    return method;
+  }
+}))
+
+const cohortRouter = require('./routes/cohorts');
+app.use('/cohorts', cohortRouter);
```

Comment:
You also need a router for the root path because when the server starts at `http://localhost:5000` there is a 404 error thrown to the user.